src/components/BookItem.js

```diff
@@ -0,0 +1,20 @@
+// import React from 'react';
+// import './BookItem.css';
+
+// const BookItem = () => {
+//   return (
+//     <li className="list-item" key={id}>
+//       <p>{title}</p>
+//       <p>{author}</p>
+//       <div className="btns">
+//         <button type="button">Comment</button>|
+//         <button type="button" onClick={() => dispatch(removeBook(id))}>
+//           Remove
+//         </button>
+//         |<button type="button">Edit</button>
+//       </div>
+//     </li>
+//   );
+// };
+
+// export default BookItem;

```

Comment:
- [ ] It is not a good practice to leave commented code, Please remove commented code from the project.
 - I can see you are not using this component `BookItem`, You should remove the component that is not being used in the project.